### ISSUE TITLE
False `E: non-readable ... 0` on %ghost files

### ISSUE THREAD

**Issue by hroncok** [rpm-software-management/rpmlint#1287]
```spec
Name:           reproducer-ghost
Version:        0
Release:        0%{?dist}
Summary:        ...
License:        MIT
BuildArch:      noarch

%description
...

%files
%ghost /boohoo
```

```console
$ rpmlint ../RPMS/noarch/reproducer-ghost-0-0.fc39.noarch.rpm
rpmlint: 2.6.1
...
reproducer-ghost.noarch: E: non-readable /boohoo 0
```

This is False error. A ghost files are not "readable". See also https://github.com/rpm-software-management/rpm/discussions/3380
